executor/tests/test_executor.py

```diff
@@ -463,3 +482,35 @@ def sleep_and_set_stop_signal():
 
         command = 'sleep 100'
         self.manage_task_runner(command, assertions, stop_signal=stop_signal)
+
+    def test_manage_task_long_output(self):
+        def assertions(driver, task_id, sandbox_directory):
+
+            logging.info('Statuses: {}'.format(driver.statuses))
+            self.assertEqual(3, len(driver.statuses))
+
+            logging.info('Messages: {}'.format(driver.messages))
+            self.assertEqual(2, len(driver.messages))
+
+            actual_encoded_message_0 = driver.messages[0]
+            expected_message_0 = {'sandbox-directory': sandbox_directory, 'task-id': task_id}
+            assert_message(self, expected_message_0, actual_encoded_message_0)
+
+            actual_encoded_message_1 = driver.messages[1]
+            expected_message_1 = {'exit-code': 0, 'task-id': task_id}
+            assert_message(self, expected_message_1, actual_encoded_message_1)
+
```

Comment:
We're missing the part where we check if the stdout file has the expected string.

Reply:
Added assertion on the string being there in the output.